src/index.js

```diff
@@ -0,0 +1,151 @@
+const state = {
+    currentTemp: 70,
+    backgroundColor: document.getElementById('buttom--left'),
+    landscapeImg: document.getElementById('buttom--right'),
+    cityInput: document.getElementById('city-input'),
+    location: null,
+    lat: null,
+    lon: null,
+    skyCondition: document.getElementById('sky--options'),
+    skyImg: document.getElementById('top')
+}
+// changes the temp on the page
+const changesTempDisplay = () => {
+    const tempControl = document.getElementById('temperature');
+    tempControl.textContent = `${state.currentTemp}`
+}
+// CHANGES TEMP
+const increaseTemp = () => {
+    state.currentTemp++;
+    changesTempDisplay()
+};
+
+const decreaseTemp = () => {
+    state.currentTemp--;
+    changesTempDisplay()
+};
+
+// sky background change with sky option
+const skyBackground = () => {
+    console.log('skyBackground')
+    const condition = state.skyCondition.value
+    if(condition == 'sunny') {
+        console.log('sunny')
```

Comment:
Remove the debugging print statements on line 30 and 33 before you open a pull request for your code to be reviewed